terminus-ui/select/src/select.component.spec.ts

```diff
@@ -1174,6 +1174,33 @@ describe(`TsSelectComponent`, function() {
       jest.runAllTimers();
     });
 
+    describe(`required`, () => {
+
+      test(`should set required if the form control is required`, () => {
+        const fixture = createComponent(testComponents.ValidateOnChange);
+        fixture.detectChanges();
+        const component = getSelectElement(fixture);
+        const selectTrigger = getSelectTriggerElement(fixture);
+        dispatchMouseEvent(selectTrigger, 'click');
+        dispatchMouseEvent(component, 'click');
+        const validationMessage = getValidationMessageElement(fixture);
+
+        expect(validationMessage).toBeTruthy();
+      });
+
+      test(`should set required if the form control is required`, () => {
```

Comment:
These two tests have the same description (lines 1177 + 1191), so clarify
`should set required if the form control is required with autocomplete`